src/main/java/no/nav/pto/veilarbportefolje/cv/CVServiceFromAiven.java

```diff
@@ -0,0 +1,41 @@
+package no.nav.pto.veilarbportefolje.cv;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import no.nav.common.types.identer.AktorId;
+import no.nav.pto.veilarbportefolje.cv.dto.CVMelding;
+import no.nav.pto.veilarbportefolje.elastic.ElasticServiceV2;
+import org.springframework.stereotype.Service;
+
+import static no.nav.pto.veilarbportefolje.cv.dto.Ressurs.CV_HJEMMEL;
+
+
+@RequiredArgsConstructor
+@Service
+@Slf4j
+public class CVServiceFromAiven {
+    private final ElasticServiceV2 elasticServiceV2;
+    private final CvRepository cvRepository;
+
+    public void behandleKafkaMelding(CVMelding melding) {
+        AktorId aktoerId = melding.getAktoerId();
+
+        if (melding.getRessurs() != CV_HJEMMEL) {
+            log.info("Ignorer melding for ressurs {} for bruker {}", melding.getRessurs(), aktoerId);
+            return;
+        }
+
+        if (melding.getSlettetDato() != null) {
```

Comment:
maybe flip this if logic? "stay positive"

Reply:
Done :)

Reply:
It does not appear to have changed

Reply:
Just pushed that change..